src/main/resources/assets/js/test/interviewer/interviewerDetailsControllerTest.js

```diff
@@ -64,20 +63,31 @@ define(['angular', 'angularMocks', 'application/interviewer/controllers/tdprInte
                     dummyData: 42,
                     bandLevel: 9001,
                     defaultStartHour: '05:15:01',
-                    defaultFinishHour: '22:49:31'
+                    defaultFinishHour: '22:49:31',
+                    password: null
                 });
             });
 
             it('should show notification on success', function () {
                 updatePersonDeferred.resolve({});
-                spyOn(Notification, 'success');
                 $scope.person = {};
 
                 $scope.updateDetails();
                 $scope.$apply();
 
                 expect(Notification.success).toHaveBeenCalledWith('Details updated!');
             });
+
+            it('should show notification when trying to save invalid password', function(){
```

Comment:
Perhaps it would be worth to add additional tests for when passwords differ and a successful scenario?